### ISSUE TITLE
Whenever I rerun my PAT model, I am getting a different error.

### ISSUE THREAD

**Issue by rwadhwa910** [NREL/OpenStudio-PAT#156]
I have got 4 different errors while running the PAT model. Every time I get a different error and the simulation fails. I have attach four screenshot each depicting a different error. 
![PAT error](https://user-images.githubusercontent.com/63880759/85780238-a166ff00-b6e1-11ea-8d6f-61b9564edcc0.PNG)
![PAT error1](https://user-images.githubusercontent.com/63880759/85780243-a2982c00-b6e1-11ea-9c21-70ff0239e347.PNG)
![PAT error2](https://user-images.githubusercontent.com/63880759/85780245-a330c280-b6e1-11ea-9108-06d3174b68b4.PNG)
![PAT error3](https://user-images.githubusercontent.com/63880759/85780249-a3c95900-b6e1-11ea-9a6b-6320c1b55cce.PNG)

**Comment by DavidGoldwasser** [comment]
Thanks @rwadhwa910 thanks for documenting this. Don't have an answer for this now, but we have seen evidence that OSW files when run multiple times with OpenStudio 3.0 give different results, with model objects being passed in, in a different order. As a note at least on one of these I notice that you are using PAT project with "RaceToZero" instead of the "sddc" one. That one is older and has not been updated. It should still fail the same way everytime, but I just wanted to confirm that all of these are from that same "RaceToZero" project and not a mix of "RaceToZero" and "sddc"

**Comment by DavidGoldwasser** [comment]
I'll need to try and reproduce this since I can't really see what the failure is on many of them. The last change building location one looks like outdated resource file; @rwadhwa910 did you do any measure updates from BCL between runs. I'm using all repo measure (which is what is on the repo branch)
https://github.com/DavidGoldwasser/PAT_projects/tree/os30_upgrade

**Comment by rwadhwa910** [comment]
> Thanks @rwadhwa910 thanks for documenting this. Don't have an answer for this now, but we have seen evidence that OSW files when run multiple times with OpenStudio 3.0 give different results, with model objects being passed in, in a different order. As a note at least on one of these I notice that you are using PAT project with "RaceToZero" instead of the "sddc" one. That one is older and has not been updated. It should still fail the same way everytime, but I just wanted to confirm that all of these are from that same "RaceToZero" project and not a mix of "RaceToZero" and "sddc"

@DavidGoldwasser These errors that I have put here are from PAT-SDDC_Primaryschool project.

> I'll need to try and reproduce this since I can't really see what the failure is on many of them. The last change building location one looks like outdated resource file; @rwadhwa910 did you do any measure updates from BCL between runs. I'm using all repo measure (which is what is on the repo branch)
> https://github.com/DavidGoldwasser/PAT_projects/tree/os30_upgrade

@DavidGoldwasser Yes I am using the repo branch.https://github.com/NREL/OpenStudio-PAT/issues/156

**Comment by rwadhwa910** [comment]
> > Thanks @rwadhwa910 thanks for documenting this. Don't have an answer for this now, but we have seen evidence that OSW files when run multiple times with OpenStudio 3.0 give different results, with model objects being passed in, in a different order. As a note at least on one of these I notice that you are using PAT project with "RaceToZero" instead of the "sddc" one. That one is older and has not been updated. It should still fail the same way everytime, but I just wanted to confirm that all of these are from that same "RaceToZero" project and not a mix of "RaceToZero" and "sddc"
> 
> @DavidGoldwasser These errors that I have put here are from PAT-SDDC_Primaryschool project.
> 
> > I'll need to try and reproduce this since I can't really see what the failure is on many of them. The last change building location one looks like outdated resource file; @rwadhwa910 did you do any measure updates from BCL between runs. I'm using all repo measure (which is what is on the repo branch)
> > https://github.com/DavidGoldwasser/PAT_projects/tree/os30_upgrade
> 
> @DavidGoldwasser Yes I am using the repo branch. 9https://github.com/DavidGoldwasser/PAT_projects/tree/os30_upgrade)

**Comment by rwadhwa910** [comment]
> I'll need to try and reproduce this since I can't really see what the failure is on many of them. The last change building location one looks like outdated resource file; @rwadhwa910 did you do any measure updates from BCL between runs. I'm using all repo measure (which is what is on the repo branch)
> https://github.com/DavidGoldwasser/PAT_projects/tree/os30_upgrade

@DavidGoldwasser  Yeah, I am using all the latest repo measure from your personal repository.
